src/main/kotlin/nl/info/zac/flowable/bpmn/model/ZaaktypeBpmnProcessDefinition.kt

```diff
@@ -44,4 +44,13 @@ class ZaaktypeBpmnProcessDefinition {
 
     @Column(name = "productaanvraagtype")
     var productaanvraagtype: String? = null
+
+    /**
+     * Optional name of the group to assign when starting a BPMN process via the Dimpact productaanvraag flow.
```

Comment:
Why is it optional? You correctly mention that it fails when it does not exist so why not make it mandatory in the first place? I expect it to be mandatory, also in the datamodel.

Reply:
Is the idea that it is only temporarily optional until we have added groups in our database on TEST or something? But then I would expect a JIRA task to make it mandatory afterwards. Do you know @hsiliev ?

Reply:
I would make it mandatory, and before merging this PR simply delete the few existing `ZaaktypeBpmnProcessDefinition` records from our database on TEST and afterwards simply recreate them (with groups).